src/components/Table/DrillDown/helpers/utils.tsx

```diff
@@ -0,0 +1,13 @@
+import { Column } from 'react-table';
+
+/** Takes an object and returns columns that are compatible with react-table
+ * derived from the object's keys
+ */
+export function columnsFromObject<D extends object>(items: D[]): Array<Column<D>> {
```

Comment:
Now this makes sense as a helper to me!

Reply:
ok, cool I will move the components to components